media-libs/imlib2/imlib2-1.4.9.recipe

```diff
@@ -0,0 +1,120 @@
+SUMMARY="An advanced replacement library for libraries like libXpm"
+DESCRIPTION="A library that does image file loading and \
+saving as well as rendering, manipulation, arbitrary polygon support, etc.
+
+It does ALL of these operations FAST. Imlib2 also tries to be highly \
+intelligent about doing them, so writing naive programs can be done \
+easily, without sacrificing speed.
+
+This is a complete rewrite over the Imlib 1.x series. The architecture is \
+more modular, simple, and flexible. See index.html in the doc/ directory \
+for more information."
+HOMEPAGE="https://www.enlightenment.org/"
+SOURCE_URI="http://downloads.sourceforge.net/project/enlightenment/imlib2-src/$portVersion/imlib2-$portVersion.tar.bz2"
+CHECKSUM_SHA256="7d2864972801823ce44ca8d5584a67a88f0e54e2bf47fa8cf4a514317b4f0021"
+REVISION="1"
+LICENSE="IMLIB2"
+COPYRIGHT="2002 Project Enlightenment"
+
+ARCHITECTURES="?x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	imlib2$secondaryArchSuffix = $portVersion  compat >= 1
+	lib:libimlib2$secondaryArchSuffix = 1.4.9 compat >= 1
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgif$secondaryArchSuffix
+	lib:libid3tag$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libtiff$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgif$secondaryArchSuffix
+	devel:libid3tag$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libtiff$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:ranlib
+	"
+
+BUILD()
+{
+	libtoolize --force --copy --install
+	aclocal
+	autoconf
+	automake --add-missing
+	runConfigure ./configure --without-x
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	# remove libtool files
+	rm $libDir/*.la
+	rm $libDir/imlib2/filters/*.la
+	rm $libDir/imlib2/loaders/*.la
+
+	prepareInstalledDevelLib libImlib2
+	fixPkgconfig
+
+	# devel package
+	packageEntries devel \
+				$binDir \
+				$developDir
+}
+
+# ----- devel package -------------------------------------------------------
```

Comment:
You can remove this line and move the PROVIDES_devel and REQUIRES_devel up, between REQUIRES and BUILD_REQUIRES, as recommended in the [guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering).